kvrocks.conf

```diff
@@ -335,6 +335,10 @@ json-max-nesting-depth 1024
 # Default: json
 json-storage-format json
 
+# Whether to enable transactional mode engine::Context.
```

Comment:
Can you please expand this comment by explaining what this mode will change if enabled/disabled?
Why it's enabled by default?

Reply:
Okay, I've explained a little bit where the changes have occurred, but explaining it in the configuration file makes it easier for users to understand the purpose.

enabled by default: This is a experimental feature that is expected to enhance the consistency of kvrocks. Enabled by default in the hope of collecting more feedback.

Reply:
Thank you for your explanation, disabled by default is more user friendly.

Aside from that, I'd like to respectfully seek your advice. For experimental features, does the community have a standard experimental process? For example, are there guidelines on when a feature should graduate to become a formal one, when it should be discontinued, and how we can quickly gather feedback?

From a developer's perspective, this feature has broad implications and could affect subsequent development. I'm concerned that some issues may not surface until later, making it difficult and time-consuming to either discontinue or refine the feature down the line. Could you provide any suggestions or best practices on this matter?

Reply:
@PokIsemaine I'm not sure we have a flow for experimental features. Additionally, it's hard to force users to test the experimental feature on their valuable production data :)